### PR TITLE
Add missing showcase card in CMS page

### PR DESCRIPTION
**Pull request by tomas862**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Adds helper card in cms page located in **/admin-dev/index.php/improve/design/cms-pages**
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | Check the page how it looks like - also when everything looks okey, try to click on 'x' to remove that block. Before the testing make sure assets are up to date by running `npm run build`

**NOTE**

**This PR is derived from https://github.com/PrestaShop/PrestaShop/pull/13163** 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13163)

### CODE REVIEW

**Comment by matks** [comment]
To be merged after https://github.com/PrestaShop/PrestaShop/pull/13234 in order to avoid conflicts

File: src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+{% block cms_pages_showcase_card %}
+  {% if not showcaseCardIsClosed %}
+    <div id="cms-pages-showcase-card" class="showcase-card card">
+      <div class="showcase-card__left shape-one helper-card__cms-pages-background img-rtl">
+        <img src="{{ asset('themes/new-theme/scss/img/helper-card/content@3x.png') }}" class="img-fluid">
+      </div>
+      <div class="showcase-card__right">
+        <h2>{{ 'Create meaningful content'|trans({}, 'Admin.Design.Help') }}</h2>
+        <p>{{ 'Because it is not just selling products but also creating a universe, build pages to tell stories andcatch your visitors’ interest and turn them into loyal customers.'|trans({}, 'Admin.Design.Help') }}</p>
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig]: Actually, I know it is slightly different from the first version but I would prefer _Because it is not just selling products but also creating a universe, build pages to tell stories and catch your visitors’ interest, to turn them into loyal customers._, seems clearer this way.

Comment (tomas862) [reply src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig]: Done :)

**Comment by matks** [comment]
@tomas862 https://github.com/PrestaShop/PrestaShop/pull/13234 is merged 👍

**Comment by tomas862** [comment]
@matks rebased :+1:

**Review by matks — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thank you @tomas862

**Pull request merged**